images/win/scripts/Installers/Install-AndroidSDK.ps1

```diff
@@ -109,21 +111,23 @@ $ndkLatestPackageName = Get-AndroidPackagesByName -AndroidPackages $androidPacka
                 | Sort-Object -Unique `
                 | Select-Object -Last 1
 
-$androidNDKs = @($ndkLTSPackageName, $ndkLatestPackageName)
+$androidNDKs = $ndkMajorVersions | Foreach-Object {
+    Get-AndroidPackagesByName -AndroidPackages $androidPackages -PrefixPackageName "ndk;$_" | Sort-Object -Unique | Select-Object -Last 1
+}
 
 Install-AndroidSDKPackages -AndroidSDKManagerPath $sdkManager `
                 -AndroidSDKRootPath $sdkRoot `
                 -AndroidPackages $androidNDKs
 
-$ndkLTSVersion = $ndkLTSPackageName.Split(';')[1]
+$ndkDefaultVersion = $ndkDefaultPackageName.Split(';')[1]
 $ndkLatestVersion = $ndkLatestPackageName.Split(';')[1]
```

Comment:
Looks a bit complicated. I think we can get rid of lines 104 and 109 as we have `$androidNDKs` variable now. These two would look like
```
$ndkDefaultVersion = ($androidNDKs | Where-Object { $_ -match "ndk;$ndkDefaultMajorVersion" }).Split(';')[1]
$ndkLatestVersion = ($androidNDKs | Where-Object { $_ -match "ndk;$ndkLatestMajorVersion" }).Split(';')[1]
```

Reply:
Updated it, thank you!